master/internal/job/job.go

```diff
@@ -178,18 +179,26 @@ func (j *Jobs) Receive(ctx *actor.Context) error {
 			switch action := update.GetAction().(type) {
 			case *jobv1.QueueControl_Priority:
 				priority := int(action.Priority)
+				if priority < 1 || priority > 99 {
+					errors = append(errors, "priority must be between 1 and 99")
+					continue
+				}
 				resp := ctx.Ask(jobActor, SetGroupPriority{
 					Priority: &priority,
 				})
 				if err := resp.Error(); err != nil {
-					errors = fmt.Sprintf("%s \n %s", errors, err.Error())
+					errors = append(errors, err.Error())
```

Comment:
Cool, so appending errors works?

Reply:
wdy mean? we still turn them into a single error before responding